### DISCUSSION TITLE
Appearance customization strategy inconsistencies

### DISCUSSION THREAD

**Discussion by sonora** [osmandapp/OsmAnd#11581] [General]
I am not a big fan of how much effort we recently put into making the appearance of every single app and UI detail hideable and customizeable.

But since we do it: What is the underlying UI philosophy? Why do we 
- in some cases place the appearance menus right where you select and work with a feature, e.g. when you tap a track or a favorite, you will find its corresponding appearance menu directly on its context card,
-  but in other cases like when you tap the route line or the position icon, the appearance configuration is not present there? Instead it is tugged away under 'Settings/Configure profile/Profile customization', a menu which by its subtitle talks about "Icon, color, and name" ?

**Comment by vshcherb** [comment]
It's hard to answer that broad question cause every single customization has own explanation. The main idea for most of it is not to target default OsmAnd users but to target custom application users i.e. there are some scenarios where the application could be customized in such sense that it could be distributed for less sophisticated users within 1 profile or may be even as a separate app.

**Reply by sonora** [reply]
Yes, probably true. I just stumble across these things every once in a while and want to bring them up to double-check if we have a reason or are just dealing with random preferences from different developers. ;)

**Reply by vshcherb** [reply]
we would never know, customization is driven by actual requests to customize the app and of course it will be always incomplete. None the less it is to create custom apps / profiles but ordinary users could enjoy customization as well.
